File: Languages/PHP/php_study_3/page7/drink.php
<?php 
require_once('menu.php');

class Drink extends Menu {
  private $type;
  
  // Salin method __construct dari class Menu 
  // Tambahkan $type ke parameter
  public function __construct($name, $price, $image, $type) {
    $this->name = $name;
    $this->price = $price;
    $this->image = $image;
    self::$count++;
    
    // Tetapkan $type ke property type
    $this->type = $type;
    
  }
  
  public function getType() {
    return $this->type;
  }
  
  public function setType($type) {
    $this->type = $type;
  }
  
}

?>

File: Languages/Ruby/ruby_study_5/page1/drink.rb
require "./menu"

class Drink < Menu
  attr_accessor :volume
  
  def initialize(name:, price:, volume:)
    super(name: name, price: price)
    self.volume = volume
  end
  
  def info
    return "#{self.name} $#{self.price} (#{self.volume}mL)"
  end
end
